### DISCUSSION TITLE
Lowest score with perfect match!

### DISCUSSION THREAD

**Discussion by KS-HTK** [cssbattle/forum#188] [General]
I got a new lowest score despite having an all black diff on #30 Horizon.
This is not a top of the Leaderboard code, so I will share it here, but it results in only 3.2% match. Which is qute funny, and frustrating.

```css
<style>*{box-shadow:0 0 0 42px#F7EC7D,0 0 0 92px#E38F66,0 0 0 192px#AA445F,0 0 0 292px#62306D
```

Why does this code get such an awful match?
Is this a bug?

**Comment by KS-HTK** [comment]
So, my code above works in firefox, but with chrome beeing the judge it breaks as in chrome the body has a hight of 100%. This is not the case in firefox.

**Reply by chinchang** [reply]
Yes. Firefox is the reason. The scoring happens on a Chromium environment.
